docs/projects/reverse-a-string/_posts/2019-10-01-javascript.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Reverse a string in JavaScript
+layout: default
+last-modified: 2019-10-01
+featured-image: <name of featured image file in assets folder>
+tags: [javascript, reverse-a-string, string-reversal]
+authors:
+  - herrfugbaum
```

Comment:
Make sure you add yourself to the author file: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml